force-app/main/default/classes/LeadTriggerHandler.cls

```diff
@@ -32,21 +32,21 @@ public with sharing class LeadTriggerHandler {
 	 */
 	public static void handleTitleNormalization(List<Lead> leadsToNormalize) {
 		for (Lead ld : leadsToNormalize) {
-			if (ld.title == 'vp' || ld.title.contains('v.p.') || ld.title.contains('vice president')) {
-				ld.Title = 'Vice President';
-			} else if (
-				ld.title.contains('mgr') ||
-				ld.title.contains('manage') ||
-				ld.title.contains('head of department')
-			) {
-				ld.Title = 'Manager';
-			} else if (ld.title.contains('exec') || ld.title == 'chief' || ld.title.contains('head')) {
-				ld.Title = 'Executive';
-			} else if (ld.title.contains('assist') || ld.title.contains('deputy') || ld.title == 'jr') {
-				ld.Title = 'Assistant';
+			// Check if the title is not null before calling any string methods
+			if (ld.Title != null) {
+				if (ld.Title == 'vp' || ld.Title.contains('v.p.') || ld.Title.contains('vice president')) {
+					ld.Title = 'Vice President';
+				} else if (ld.Title.contains('mgr') || ld.Title.contains('manage') || ld.Title.contains('head of department')) {
+					ld.Title = 'Manager';
+				} else if (ld.Title.contains('exec') || ld.Title == 'chief' || ld.Title.contains('head')) {
+					ld.Title = 'Executive';
+				} else if (ld.Title.contains('assist') || ld.Title.contains('deputy') || ld.Title == 'jr') {
+					ld.Title = 'Assistant';
```

Comment:
Try to use the containsIgnoreCase() method just in case you get 'Manage' instead of 'manage'